odxtools/readdiagcommconnector.py

```diff
@@ -0,0 +1,86 @@
+# SPDX-License-Identifier: MIT
+from dataclasses import dataclass
+from typing import Any
+from xml.etree import ElementTree
+
+from .diagcomm import DiagComm
+from .exceptions import odxrequire
+from .odxdoccontext import OdxDocContext
+from .odxlink import OdxLinkDatabase, OdxLinkId, OdxLinkRef
+from .readparamvalue import ReadParamValue
+from .snrefcontext import SnRefContext
+
+
+@dataclass(kw_only=True)
+class ReadDiagCommConnector:
+    read_param_values: list[ReadParamValue]
+
+    # exactly one of the following attributes must be non-None
+    read_diag_comm_ref: OdxLinkRef | None = None
+    read_diag_comm_snref: str | None = None
+
+    # exactly one of the following attributes must be non-None
+    read_data_snref: str | None = None
+    read_data_snpathref: str | None = None
+
+    @property
+    def read_diag_comm(self) -> DiagComm | None:
+        return self._read_diag_comm
+
+    # READ-DATA cannot be resolved ahead of time as it points to a
+    # value parameter of the positive responses that are received as
+    # an answer after executing the read diag comm...
+    #@property
+    #def read_data(self) -> ValueParameter:
+    #    return self._read_data
```

Comment:
Commented out code?

Reply:
this is meant as a reminder that this should be resolved, but at load time this is impossible as far as I can see...

Reply:
a ticket would be better reminder

Reply:
agreed. fixed: #431, https://github.com/mercedes-benz/odxtools/pull/428/commits/d3106d87e22ecd73f5557397ce4d58667e89b12a